test/basic.bats

```diff
@@ -304,3 +304,15 @@ EOF
     [ ! -f dest/rootfs/favicon.ico ]
     [ ! -d dest/rootfs/stacker ]
 }
+
+@test "commas in substitute flags ok" {
+    cat > stacker.yaml <<EOF
+busybox:
+    from:
+        type: oci
+        url: $BUSYBOX_OCI
+    run: |
+        touch /foo
+EOF
+    stacker build --substitute "a=b,c"
```

Comment:
How is this substitution used though in the stacker.yaml?

Reply:
Do you mean in the testcase?  It's not.  That's not necessary to trigger the regression failure-to-build in main branch.

Reply:
Hm. When I run this test locally (sudo bats test/basic.bats), i get zero failures.

Reply:
No, I was wrong - I can reproduce this.  No idea why.

Reply:
The CI failure is due to the gzip issue. It is a concern, but unrelated to this PR.

Reply:
Actually it seems to just be a flaky test? :   

```
ubuntu@stacker:~/stacker$ sudo -E bats test/gzip.bats 
 ✓ import various sizes 

1 test, 0 failures
```